src/Scripting/CSharp/CSharpScriptingResources.resx

```diff
@@ -125,9 +125,7 @@
   </data>
   <data name="InteractiveHelp" xml:space="preserve">
     <value><![CDATA[Usage: csi [option] ... [script-file.csx] [script-argument] ...
-
 Executes script-file.csx if specified, otherwise launches an interactive REPL (Read Eval Print Loop).
```

Comment:
Blank lines were removed.

Reply:
force pushed again - sorry for the trouble